README.md

```diff
@@ -1,24 +1,47 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Microverse Re-former Project
 
-Things you may want to cover:
+> This project will give us a chance to actually build some forms, both using nearly-pure HTML and then graduating to using the helper methods that Rails provides.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Rails
 
-* Configuration
+## Getting Started
+### To get a local copy run the following steps:
 
-* Database creation
+- [Copy this link](git@github.com:Mhdez221993/re-former.git)
+- Open your terminal or command line
+- Run `git clone` and Paste the link
+- Open the folder with your code editor
+- Run `bundle install` to install all dependecies
+- Run `rails db:migrate` to create the database tables
+- Run `rails console` to test the project
```

Comment:
The detailed instructions for running the app are currently incomplete.  Let me add some instructions. Continuing from where you stopped:
Before `rails console`. First I need to test if the app is working as expected using 

- `rails server`
- `http://localhost:3000/users/new` to create a new user
- `http://localhost:3000/users/1/edit` to edit an existing user

To confirm this:

- Run `rails console`
- `User.all` to display the exiting user.

You can refactor the above instructions to produce something similar
Lastly, I couldn't run the app without running `yarn install`. So do well to include it in the instructions